test/WebJobs.Script.Tests/EndToEndTestsBase.cs

```diff
@@ -69,6 +69,18 @@ public EndToEndTestsBase(TTestFixture fixture)
             Assert.Equal(doc.Id, id);
         }
 
+        protected async Task NotificationHubTest()
+        {
```

Comment:
comment here that there's no actual validation done b/c it's hard (impossible?) to verify from within this project. Instead, we're making sure the function is called and exits without throwing.